WiserTaskScheduler/WiserTaskScheduler/Program.cs

```diff
@@ -93,6 +98,9 @@ private static void ConfigureWtsServices(IServiceCollection services, HostBuilde
             services.AddScoped<IAccountsService, AccountsService>();
             services.AddScoped<IBranchesService, BranchesService>();
             services.AddScoped<IRolesService, RolesService>();
+            services.AddScoped<IFtpHandlerFactory, FtpHandlerFactory>();
+            services.AddScoped<FtpsHandler>();
+            services.AddScoped<SftpHandler>();
```

Comment:
Klopt het dat je hier maar 1 parameter hebt gebruikt? Bij alle andere `AddScoped` heb je eerst de interface en daarna de service, bij deze 2 heb je alleen de service/handler.

Reply:
Zoals besproken gebruiken beide handlers dezelfde interface en moet de precieze waarde gebruikt worden. De factory in de GCL vraagt ook expliciet de juiste handler op.